std/regex/package.d

```diff
@@ -209,7 +209,8 @@ $(TR $(TD Objects) $(TD
   $(REG_START Character classes )
   $(REG_TABLE
     $(REG_TITLE Pattern element, Semantics )
-    $(REG_ROW Any atom, Has the same meaning as outside of a character class.)
+    $(REG_ROW Any atom, Has the same meaning as outside of a character class. 
+      Except for ] which has to be written as \])
```

Comment:
> Except for `]` which has to be written as `\]`)

How does this render? I expect that it would be a good idea to encase `]` and `\]` in backticks. the backlash may need additional escaping.

Also `which has to be written as` should be `which must be written as` and that row of text should be one sentence: `... character class, except for...`

Reply:
Thank you for your comments!

Is there an easy way to view the docs from the CI build?

Reply:
click on the `details` of the CyberShadow/DAutoTest, then click base result : view, then click phobos and navigate to the page of interest.

Reply:
It renders ok now.